src/solvers/imex_ark.jl

```diff
@@ -80,16 +80,12 @@ function step_u!(integrator, cache::IMEXARKCache)
     s = length(b_exp)
 
     if !isnothing(T_imp!)
-        update!(
-            newtons_method,
-            newtons_method_cache,
-            NewTimeStep(t),
-            jacobian ->
-                isnothing(γ) ? error("The tableau does not specify a unique value of γ for the \
-                                      duration of each time step; do not update based on the \
-                                      NewTimeStep signal when using this tableau.") :
-                T_imp!.Wfact(jacobian, u, p, dt * γ, t),
-        )
+        (; update_j_cache, j) = newtons_method_cache
+        if !isnothing(j)
```

Comment:
The error message could be added back in, or we could pass gamma to the function and define two step_u methods (where one errors)